controllers/my_goal.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const pool = require("../db.js");
+
+router.get("/", async (req, res, next) => {
+  const nickname = req.session.user["nickname"];
+  const my_goal = await pool.query(`SELECT * FROM goal WHERE goal_user = ?`, [
+    nickname,
+  ]);
```

Comment:
```suggestion
  try{
   const my_goal = await pool.query(`SELECT * FROM goal WHERE goal_user = ?`, [
    nickname,
  ]); }
  catch (err) {
    console.error(err);
  }
```
async await 문을 사용하기 때문에 try-catch문을 통해 예외처리를 해주는 것이 좋을 것 같습니다.

Reply:
확인 후 수정했습니다.